fast64_internal/panels.py

```diff
@@ -0,0 +1,46 @@
+
+import bpy
+from .fast64_internal import *
+
+sm64GoalImport = 'Import' # Not in enum, separate UI option
+sm64GoalTypeEnum = [
+	('All', 'All', 'All'),
+	('Export Object/Actor/Anim', 'Export Object/Actor/Anim', 'Export Object/Actor/Anim'),
+	('Export Level', 'Export Level', 'Export Level'),
+	('Export Displaylist', 'Export Displaylist', 'Export Displaylist'),
+	('Export UI Image', 'Export UI Image', 'Export UI Image'),
+]
+
+class SM64_Panel(bpy.types.Panel):
+    bl_space_type = 'VIEW_3D'
+    bl_region_type = 'UI'
+    bl_category = 'SM64'
+    bl_options = {'DEFAULT_CLOSED'}
```

Comment:
i can switch it, but dont you think that most panels should be closed by default? I thought having it be a forced override would make it so that no new panels are open by mistake, keeping everything clean